src/mlpack/methods/hmm/hmm.hpp

```diff
@@ -290,6 +290,75 @@ class HMM
    */
   double LogLikelihood(const arma::mat& dataSeq) const;
 
+  /**
+   * Compute the log of the scaling factor of the given emission probability
+   * at time t. To calculate the log-likelihood for the whole sequence,
+   * accumulate log scale over the entire sequence
+   * This is meant for incremental or streaming computation of the
+   * log-likelihood of a sequence. For the first data point, provide an empty
+   * forwardLogProb vector.
+   *
+   * @param emissionLogProb emission probability at time t.
+   * probability up to time t-1
```

Comment:
```suggestion
```
I think that this line was leftover from when we passed `logLikelihood` as a parameter.  Correct me if I'm wrong. :+1: